torch/ao/quantization/quantize.py

```diff
@@ -62,11 +62,6 @@ def get_default_custom_config_dict():
     """
     return _DEFAULT_CUSTOM_CONFIG_DICT
 
-def is_activation_post_process(module):
```

Comment:
I saw that the observer [function](https://github.com/pytorch/pytorch/blob/133e61af7a8f4b098daf7d34f848e3c2a6cb4ae4/torch/ao/quantization/observer.py#L1437) and this different, just wanted to double check

Reply:
yeah, it added a few definitions that weren't captured so it will be more accurate unless people were using this as a negative test.